src/components/plots/barPlot.ts

```diff
@@ -179,12 +179,21 @@ export module Plot {
           var secondaryDist = 0;
 
           // if we're inside a bar, distance in both directions should stay 0
-          if (!_Util.Methods.intersectsBBox(queryPoint.x, queryPoint.y, barBBox)) {
+          if (!_Util.Methods.intersectsBBox(queryPoint.x, queryPoint.y, barBBox, 0)) {
             var plotPtPrimary = this._isVertical ? plotPt.x : plotPt.y;
-            var plotPtSecondary = this._isVertical ? plotPt.y : plotPt.x;
-
             primaryDist = Math.abs(queryPtPrimary - plotPtPrimary);
-            secondaryDist = Math.abs(queryPtSecondary - plotPtSecondary);
+
+            // compute this bar's min and max along the secondary axis
+            var barMinSecondary = this._isVertical ? barBBox.y : barBBox.x;
+            var barMaxSecondary = barMinSecondary + (this._isVertical ? barBBox.height : barBBox.width);
+
+            if (queryPtSecondary > barMinSecondary && queryPtSecondary <= barMaxSecondary) {
```

Comment:
So, my intent was to tie-break using this approach, but after thinking about it again, I'm not sure. _intersectsBBox() will end up picking two bars if you're exactly on their border value (note the >= and <= in its comparison code) so perhaps it's reasonable to do the same thing here and let the user deal with multiple entities exactly the same distance apart.

 so it's reasonable that this logic do the same and leave it up to the user to deal with two